jekyll-sitemap.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r!^(test|spec|features)/!)
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "jekyll", "~> 3.3"
+  spec.add_dependency "jekyll", "~> 3.4", ">= 3.4.2"
```

Comment:
Do we want to say the plugin *requires* 3.4.2? Or is it that it gets additional functionality with 3.4.2?